femtools/Field_Options.F90

```diff
@@ -1337,4 +1337,25 @@ function extract_prognostic_velocity(state, stat) result(vfield)
 
   end function extract_prognostic_velocity
 
+  subroutine get_surface_ids(bc_path, mesh, surface_ids)
```

Comment:
Could you add a one line comment saying exactly what this routine does (or more if required).